### DISCUSSION TITLE
Serving Size question

### DISCUSSION THREAD

**Discussion by 813BoZai** [PaddlePaddle/PaddleOCR#13396] [Q&A]
“{'error': 'Received message larger than max (4834729 vs. 4194304)', 'code': 8, 'message': 'Received message larger than max (4834729 vs. 4194304)', 'details': []},the size of image is (4032, 3024, 3)” what is up? about this question?

**Comment by UserWangZz** [comment]
图像分辨率太大了，尝试降低分辨率试试

**Reply by 813BoZai** [reply]
有那块地方能把这个给调大一些？

**Reply by UserWangZz** [reply]
能具体说说这个错误信息吗？

**Reply by 813BoZai** [reply]
我现在想测试Serving模式下的paddle，想知道local模式下->"det_max_side_len": 960,与Serving模式的那个字段匹配，以及我该怎么去测试，感谢

**Reply by UserWangZz** [reply]
只需要使用server模型即可
